### DISCUSSION TITLE
Missing flatnode file

### DISCUSSION THREAD

**Discussion by sdrll** [osm-search/Nominatim#3386] [General]
Hi, I've recently removed the flatnodefile from the world import. However, I now realize that I need it to refresh the Nominatim data. Could I recreate it to fulfill this requirement without importing world data again? or can i refresh it without the flatnode file ?

**Comment by mtmail** [comment]
That's not possible. It's the same as if you deleted a required database table. Sadly you'd need to do a full import again.

**Reply by sdrll** [reply]
Thank you for your answer. Actually, I found an older flatnode file that matches an import I have done 1 month before the current state of my database. Is this an issue?

**Reply by mtmail** [reply]
Only if that was also the date of the last database update (check the import_status table or run /status?format=json). If the database contains newer data then they’re out of sync and you risk bad data to creep in with each data update.
